code/modules/mob/living/carbon/human/life.dm

```diff
@@ -61,13 +64,27 @@
 	return pressure
 
 
+/mob/living/carbon/human/proc/handle_psychotherapist()
+	for(var/mob/living/carbon/human/H in in_range(src,3))
+		if(H == src)
+			continue
+		var/datum/component/mood/mood = H.GetComponent(/datum/component/mood)
+		mood.adjustPsychInstability(1)
```

Comment:
this should be done in response to the signal, this is violating the abstractions of the mood component